app/steps/ui/summary/template.html

```diff
@@ -36,7 +36,7 @@ <h2 class="heading-medium">{{content.Summary.deceasedHeading}}</h2>
     </dl>
 
     <p><img class="icon-file-download" alt="Download PDF document"/>&nbsp;
-        <a id="checkAnswerHref" href="/checkAnswersPdf">{{content.Summary.checkAnswersPdf}}</a></p>
+    <a id="checkAnswerHref" href="/checkAnswersPdf">{{content.Summary.checkAnswersPdf}}</a></p>
```

Comment:
Shouldn't we be using a feature toggle for this link?

Reply:
I refer to my earlier answer on deployment of the business service before the front end.